### PR TITLE
Removed dated WinForms content

### PR DESCRIPTION
**Pull request by rpetrusha**
## Removed dated WinForms content

Related to #14879

### CODE REVIEW

**Review by Thraka — APPROVED** [review]
Sorry about that Ron, GitHub didn't refresh and show me you had a pending review. I had a tab to that PR open from earlier in the day.

**Review by Thraka — APPROVED** [review]
Sorry about that Ron, GitHub didn't refresh and show me you had a pending review. I had a tab to that PR open from earlier in the day.

**Comment by Youssef1313** [comment]
I think you mean #14879, not #14878

File: docs/framework/winforms/advanced/encoding-and-windows-forms-globalization.md [hunk 1]
```diff
@@ -26,9 +26,7 @@ Windows Forms applications are entirely Unicode-enabled, meaning that each chara
...
+The benefits of Unicode-enabled forms include the ability to work with scripts that are Unicode-only, such as Hindi. In addition, you can use multiple languages on a single form. In Unicode, all characters are two bytes long, so no special effort is needed to represent double-byte characters. You can also write a single set of code that will work on all platforms.
 
 For Unicode-aware alternatives to the <xref:System.Windows.Forms.ToolBar> and <xref:System.Windows.Forms.StatusBar> controls, use the <xref:System.Windows.Forms.ToolStrip> and <xref:System.Windows.Forms.StatusStrip> controls, which replace these older controls. To maintain a similar look and feel between visual elements in your application, use the <xref:System.Windows.Forms.MenuStrip> control for rendering menus instead of <xref:System.Windows.Forms.MainMenu>. Like <xref:System.Windows.Forms.ToolStrip> and <xref:System.Windows.Forms.StatusStrip>, <xref:System.Windows.Forms.MenuStrip> can also process and display Unicode characters.
```
Comment (Youssef1313) [inline docs/framework/winforms/advanced/encoding-and-windows-forms-globalization.md]: Isn't this paragraph related to the deleted one ?
The deleted one mentions some controls that don't support unicode in win98, this one shows alternatives to those controls.

Comment (Thraka) [reply docs/framework/winforms/advanced/encoding-and-windows-forms-globalization.md]: Yeah, I feel like that should be removed too. But I also feel like this article has no longer offers anything of value too.

Comment (rpetrusha) [reply docs/framework/winforms/advanced/encoding-and-windows-forms-globalization.md]: Thanks for pointing that our, @Youssef1313; I'd completely missed it. And I agree, @Thraka; the topic offers nothing of value. It was clearly written for .NET Framework 1.0 to explain why Unicode was suddenly being used. I think that's become a non-issue.

File: docs/framework/winforms/advanced/international-fonts-in-windows-forms-and-controls.md [hunk 2]
```diff
@@ -80,5 +80,4 @@ textBox1.Font = new System.Drawing.Font(this.Font, textBox1.Font.Style);
   
 ## See also
 
-- [Globalizing Windows Forms applications](globalizing-windows-forms.md)
```
Comment (Thraka) [inline docs/framework/winforms/advanced/international-fonts-in-windows-forms-and-controls.md]: @rpetrusha I found this one and I removed it.

**Review by Thraka — APPROVED** [review]
I found another entry and removed it. I committed directly to your branch.

File: docs/framework/winforms/advanced/international-fonts-in-windows-forms-and-controls.md (hunk 2 above)
Comment (rpetrusha) [reply docs/framework/winforms/advanced/international-fonts-in-windows-forms-and-controls.md]: Thanks, @Thraka.

**Pull request merged**